pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -82,6 +84,11 @@ mkCommon type rec {
     mv $out/ThirdPartyNotices.txt $out/share/doc/$pname/$version/
 
     ln -s $out/dotnet $out/bin/dotnet
+    # completions snippets taken from https://learn.microsoft.com/en-us/dotnet/core/tools/enable-tab-autocomplete
```

Comment:
could we do this in `pkgs/development/compilers/dotnet/common.nix` to support the source-built things as well?

Reply:
:+1:  I think we can

Reply:
I moved installShellFiles call to common.nix and tested that completions still works for dotnet_sdk package, as well as dotnetCorePackages.dotnet_8.sdk (which looks like the source-built package you've mentioned) .